js/platform.js

```diff
@@ -116,6 +165,26 @@ Platform.prototype.draw = function (ctx) {
     Entity.prototype.draw.call(this, ctx);
 };
 
+Platform.prototype.drawWithRanking = function (ctx, goats) {
```

Comment:
Refer to other comment about Goats/Platforms; this should either be encapsulated into endgame.js or taken out altogether -> simply draw the platform images and manipulate their x,y positions in update()